### DISCUSSION TITLE
TCPServer无线环境下会出现客户端掉线后重连不上的问题，请问要从哪里去找原因呢，此时能PING通

### DISCUSSION THREAD

**Discussion by yuzhenboy** [chenshuo/muduo#570] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

If you can't compile muduo, make sure you install `cmake` and `boost` from the
official package repository, e.g. `apt` or `yum`, before opening a bug.
Don't open a bug if you installed boost from a third-party source or
downloaded it by yourself, and couldn't compile muduo, thank you.

Also specify the exact environment where the issue occurs:

**Comment by chenshuo** [comment]
先 tcpdump 分别在 client 和 server 抓包看一看？
